tokens/connect/nifty.md

```diff
@@ -69,7 +69,7 @@ description: 此教程将会带您了解如何连接Nifty钱包，一个基于
 
 ## 使用Nifty钱包 {: #using-nifty-wallet }
 
-Nifty钱包为Web3工具的提供者，如[Remix](/builders/tools/remix/)。通过将Nifty钱包连接至Moonbeam网络之一，您可以如同在MetaMask上部署合约般部署合约，并且通过Nifty签署交易。本教程会使用Moonbase Alpha测试网，但也适用于Moonbeam和Moonriver。
+Nifty钱包为Web3工具的提供者，如[Remix](/builders/build/eth-api/dev-env/remix/)。通过将Nifty钱包连接至Moonbeam网络之一，您可以如同在MetaMask上部署合约般部署合约，并且通过Nifty签署交易。本教程会使用Moonbase Alpha测试网，但也适用于Moonbeam和Moonriver。
```

Comment:
line 78 should use the latest URL /builders/get-started/networks/moonbase/#get-tokens/